systemtest/src/main/java/io/strimzi/systemtest/performance/PerformanceConstants.java

```diff
@@ -176,6 +186,6 @@ public interface PerformanceConstants {
     /**
      * Performance specific related tags
      */
-    String PERFORMANCE_CAPACITY = "performance-capacity";
+    String CAPACITY = "capacity";
```

Comment:
> Guess this should be next to the others tags in case it is used as `@Tag()`. I dont say you have to move it in this PR, but. maybe it might be reasonable to create something like `Tags.java` with tags related constatns. Or just moves it to `Constants`

Noted. I can do it in the the next PR :) Dedicated class for Tags.